brands/shop/supermarket.json

```diff
@@ -4001,6 +4001,15 @@
       "name": "Superspar",
       "shop": "supermarket"
     }
+  },
+    "shop/supermarket|Super del Norte": {
+    "countryCodes": ["mx"],
+    "tags": {
+      "brand": "Super del Norte",
+      "brand:wikidata": "Q88388513Q88388513",
```

Comment:
Could you fix the wikidata to have all in one commit as suggested by @Adamant36 ?

Best.

Reply:
Idon't have the permissions to do that